rhel/SOURCES/percona-qan-api.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Percona Query Analytics API
-After=network.target mysqld.service
-Requires=mysqld.service
+After=network.target postgresql.service
+Requires=postgresql.service
```

Comment:
It requires ClickHouse, but not PostgreSQL.

Reply:
Fixed